config/toml.go

```diff
@@ -156,6 +156,7 @@ priv_validator_state_file = "{{ js .BaseConfig.PrivValidatorState }}"
 
 # TCP or UNIX socket address for Ostracon to listen on for
 # connections from an external PrivValidator process
+# If this value is set, key file(priv_validator_key.json) will not be generated.
```

Comment:
Is the default value of `priv_validator_laddr` empty string?

Reply:
Yes, default value is empty string.

Reply:
Actually, default value comes from {{ .BaseConfig.PrivValidatorListenAddr }}. which is empty string though.

Reply:
If it is empty string, how about adding the default value explanation also?